### DISCUSSION TITLE
使用内置MdCatalog组件怎么实现目录

### DISCUSSION THREAD

**Discussion by 1256759340** [imzbf/md-editor-v3#718] [Q&A]
目前按照示例上面的引用只是加载出来目录但是没有点击效果

**Comment by imzbf** [comment]
参考：https://github.com/imzbf/md-editor-v3/blob/dev-docs/layouts/Catalog/index.vue

**Reply by 1256759340** [reply]
实现有参考那个示例，目录也能正确显示出来，但是目录没点击效果，是需要在另外加其他事件吗

**Reply by imzbf** [reply]
如果你说的是点击后，文章的标题能够跑到顶部的话，这个是需要你自行实现的，除非你的项目是SSR项目
